pkg/helm/client/client.go

```diff
@@ -98,16 +101,21 @@ func NewRESTClientGetter(mgr manager.Manager, ns string) (genericclioptions.REST
 
 var _ kube.Interface = &ownerRefInjectingClient{}
 
-func NewOwnerRefInjectingClient(base kube.Client, ownerRef metav1.OwnerReference) kube.Interface {
+func NewOwnerRefInjectingClient(base kube.Client, ownerRef metav1.OwnerReference,
+	mgr manager.Manager, cr *unstructured.Unstructured) kube.Interface {
```

Comment:
Pass the REST mapper in directly since we don't need the whole manager.
```suggestion
	restMapper rest.Mapper, cr *unstructured.Unstructured) kube.Interface {
```